Source/Core/Util/EncounterSlot.cpp

```diff
@@ -100,7 +100,9 @@ namespace EncounterSlot
             return calcSlot<5>(compare, std::array<u8, 5> { 60, 90, 95, 99, 100 });
         case Encounter::BugCatchingContest:
             return calcSlot<10, true>(compare, std::array<u8, 10> { 80, 60, 50, 40, 30, 20, 15, 10, 5, 0 });
-        case Encounter::HeadButt:
+        case Encounter::SafariZone:
+            return compare % 10;
```

Comment:
Why was this added back. It's dead code right now

Reply:
Removed

Reply:
This was my bad, somehow was leftover from merging in updated items